packages-web/calendar/typings/Calendar.d.ts

```diff
@@ -2,7 +2,6 @@ type VisibilityMap<T> = {
     [P in keyof T]: any;
 };
 
-declare module "moment";
```

Comment:
We dont need this anymore ?

Reply:
Nope, the library provides the d.ts inside and we were basically overridden